build_tools/github/upload_anaconda.sh

```diff
@@ -11,12 +19,10 @@ else
     ANACONDA_TOKEN="$SCIKIT_LEARN_STAGING_UPLOAD_TOKEN"
 fi
 
-# Install Python 3.8 because of a bug with Python 3.9
-export PATH=$CONDA/bin:$PATH
-conda create -n upload -y python=3.8
-source activate upload
-conda install -y anaconda-client
-
-# Force a replacement if the remote file already exists
-anaconda -t $ANACONDA_TOKEN upload --force -u $ANACONDA_ORG dist/artifact/*
-echo "Index: https://pypi.anaconda.org/$ANACONDA_ORG/simple"
+if [ "$ANACONDA_TOKEN" == "" ];
```

Comment:
I would prefer `if [ -z "$ANACONDA_TOKEN" ]`, but the proposed solution would work anyway.